domain/consensus/processes/coinbasemanager/coinbasemanager.go

```diff
@@ -58,25 +74,30 @@ func (c *coinbaseManager) ExpectedCoinbaseTransaction(blockHash *externalapi.Dom
 	}, nil
 }
 
+func (c *coinbaseManager) daaAddedBlocksSet(blockHash *externalapi.DomainHash) (hashset.HashSet, error) {
+	daaAddedBlocks, err := c.daaBlocksStore.DAAAddedBlocks(c.databaseContext, blockHash)
+	if err != nil {
+		return nil, err
+	}
+
+	daaAddedBlocksSet := hashset.New()
```

Comment:
You can just call `hashset.NewFromSlice(daaAddedBlocks)` (which should probably be changed to preallocate)